frontend/src/app/deployable/template-editor/template-editor.component.ts

```diff
@@ -78,6 +78,7 @@ export class TemplateEditorComponent implements OnInit {
   public templatesValid: EventEmitter<boolean> = new EventEmitter<boolean>();
   public configurationTemplatesModels: Array<ConfigurationTemplateEditorModel> = [];
   public selectedTab = new FormControl(0);
+  public aceOptions = {useSoftTabs: true, navigateWithinSoftTabs: true, tabSize: 2};
```

Comment:
This might cause trouble for config maps, that technically can contain all kinds of nonesense.
We should either use this only for files that are pure yaml without config or not use this after all.

Reply:
Right. I removed the `useSoftTabs` and `navigateWIthingSoftTabs` settings. While unimportant, I guess we can keep the `tabSize` of 2 though.